ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -266,12 +268,22 @@ - (void)call:(TVOCall *)call didDisconnectWithError:(NSError *)error {
         NSLog(@"Call disconnected");
     }
 
+    if (!self.userInitiatedDisconnect) {
+        CXCallEndedReason reason = CXCallEndedReasonRemoteEnded;
```

Comment:
I noticed we are not using all end call reasons - https://developer.apple.com/documentation/callkit/cxcallendedreason